### PR TITLE
Download and Incorporate PTC Capability

### PR DESCRIPTION
**Pull request by mrmundt**
This PR has several changes:

- Move scripts to `assets` directory (with styles and images)
- Add a script and GitHub Action to insert `Download this PTC` capability to all new cards
- Create blank "Progress Tracking Card Template" for the catalog
- Add instructions with examples on how to download and incorporate PTCs into GitHub and GitLab projects
- Add a "Using the Catalog" summary on the index page

### CODE REVIEW

**Comment by mrmundt** [comment]
I have a test site of this available at: https://mrmundt.github.io/ptc-catalog/

File: catalog/ProgressTrackingCardTemplate.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+| 0 | Current state. |
+| 1 | Stage 1 state. |
+| 2 | Stage 2 state. |
+| 3 | Stage 3 state. |
+| 4 | Stage 4 state. |
+| 5 | Stage 5 state (OPTIONAL). |
+
+## Comments
+
+Any comments additional comments here.
```
Comment (killillian) [inline catalog/ProgressTrackingCardTemplate.md:30]: I think this is just meant to say "Any additional comments here."

**Review by killillian — COMMENTED** [review]
Everything looks great! I think the download links are very useful, and navigating the pages was easy and intuitive. The examples were also very helpful, especially since they documented both GitHub and GitLab GUIs. I noticed two very small typos that I commented on in-line, but otherwise no issues!

**Review by killillian — APPROVED** [review]
Looks good to me!

**Comment by mrmundt** [comment]
Ping @elaineraybourn @wtsutherland @rmmilewi @gonsie

File: catalog/ProgressTrackingCardTemplate.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+## User Story
+
+Next, provide one or more **user stories** that explain the context of this goal.
+A user story is a statement that can be phrased as "As a ____, I want ____ so that ____." 
+
+As a **ROLE**, I want **DESIRE**, so that **IMPACT**.
+
+## Card(s)
+
+| Score         | Description |
```
Comment (elaineraybourn) [inline catalog/ProgressTrackingCardTemplate.md:19]: We discussed changing "score" to "Stage" or something that does not suggest assessment. Does it make sense to change this here?

**Review by elaineraybourn — CHANGES_REQUESTED** [review]
HI @mrmundt very nice, just change the reference to score that I made in my comment and we are good to go.

**Pull request merged**